.env.sample

```diff
@@ -7,6 +7,8 @@ CHARON_LATEST_REL=                  # charon latest releaesd version
 CHARON_OLDEST_REL=                  # chaon oldest supported release version
 TEKU_VERSION=                       # teku vc image version/tag, i.e 22.6.0
 lIGTHTHOUSE_VERSION=                # lighthouse vc image version/tag, i.e 3.2.1
+LODESTAR_VERSION=                   # lodestar vc image version/tag, i.e v1.4.1
+VC_TYPES=                           # comma separated type of VC allowed for each operator, i.e 0,1,2,0,0 (0=teku, 1=lighthouse, 2=lodestar, etc)
```

Comment:
@eth2devops I have added `VC_TYPES` var to add multiple VC configuration to canary cluster. For example, for canary-goerli-1 i have added: `2,2,2,2,0,0,0` which means first four VCs are lodestar and last 3 are teku.

But this will need update to all the existing canary cluster environment variable files that are stored in gcp. (I have already updated `canary-goerli-1.env`)

Reply:
Thanks @dB2510 for the update. I will handle the rest of the clusters.